src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    protected Color color;
```

Comment:
why protected?
please make the field private and create a constructor that accepts color